src/commands/deploy/deploy.ts

```diff
@@ -118,6 +119,12 @@ const getDeployFolder = async ({
   }
 
   if (!deployFolder) {
+    if (!stdin.isTTY || !stdout.isTTY) {
+      // non interactive - can't get the value, resolve to the cwd
+      deployFolder = command.workspacePackage ? command.jsWorkspaceRoot || site.root : command.workingDir
+      return deployFolder || command.workingDir
```

Comment:
nit: I personally find the nesting of conditionals quite confusing and would prefer a more explicit version, even if more verbose. Up to you, though.

```suggestion
      if (command.workspacePackage) {
        if (command.jsWorkspaceRoot) {
          return command.jsWorkspaceRoot
        }

        if (site.root) {
          return site.root
        }
      }

      // Either not a workspace package, or empty site root.
      return command.workingDir
```